lib/sunrise/sunrise.php

```diff
@@ -67,12 +67,18 @@ function handle_not_found_error( $error_type ) {
 	$is_web_request = Context::is_web_request();
 	if ( $is_web_request ) {
 		$is_maintenance_mode = Context::is_maintenance_mode();
+		$is_overdue_locked   = Context::is_overdue_locked();
 		if ( $is_maintenance_mode ) {
 			// 503 prevents page from being cached.
 			// We handle healthchecks earlier and don't have to worry about them.
 			$status_code = 503;
 			header( 'X-VIP-Go-Maintenance: true' );
 			$error_doc = sprintf( '%s/mu-plugins/errors/site-maintenance.html', WP_CONTENT_DIR );
+		} elseif ( $is_overdue_locked && Context::is_vip_env() ) {
+			// Similar with maintenance mode, but with a different message.
+			$status_code = 503;
+			header( 'X-VIP-Go-Maintenance: true' ); // TODO: different header?
```

Comment:
Should we use a different header here? I don't think there is any disadvantage on leveraging the same header, but let me know your thoughts.

Reply:
I would say yes - the site isn't in maintenance mode so this header would be confusing / misleading.

Reply:
```suggestion
			header( 'X-VIP-402: true' );
```